js/scenes/NumberOfGuestScreen/NumberOfGuestScreenContainer.js

```diff
@@ -56,22 +70,66 @@ class NumberOfGuestScreenContainer extends React.Component {
     }
   }
 
-  increase = () => this.setState({ count: this.state.count + 1 });
+  increase = () => this.setState({ count: (this.state.count * 1) + 1 });
```

Comment:
it's a quick way to convert the count from a string to an integer so that adding 1 doesnt result in string addition.